example/example.cpp

```diff
@@ -77,6 +77,30 @@ int main() {
     assert(status == LineairDB::TxStatus::Committed);
   }
 
+  {
+    LineairDB::Database db;
+    LineairDB::TxStatus status;
+
+    db.ExecuteTransaction(
```

Comment:
Please write a brief explanation of what kind of example it is, just one line is fine.
Also, please write it for L60 as well.